YvesIraguha/lost-and-found-api
# change file storage to cloud storage

#### What does this PR do?
Change file storage from file system to cloud provider
#### Description of Task to be completed?
- removing multer functionality
- Saving only photo url in our database
#### How should this be manually tested?
- Clone this repo and switch your working dirctory to *lost-and-found-api*
- Create a `.env` and update it with `.env-sample` data
- Checkout this branch `git checkout fix/use-cloud-provider-for-userProfile`
- Install all dependencies and start the server
- In postman login to this app on `/api/v1/users/auth/login`
- Use the returned token in postman headers as `auth-token`
- Still in postman send a **POST** request on `/api/v1/users/profile/image`  with url of your profile picture
> You should see returned user with photoUrl of the picture url you sent

#### Any background context you want to provide?
- Login is performed only if you have an account on the app
- Data for login
```json
{
    "email":"email@example.com",
    "password":"password"
}
```
- Data for profile picture
```json
{
    "photoUrl":"https://photo/location"
}
```
#### What are the relevant pivotal tracker stories?
[#174143979](https://www.pivotaltracker.com/story/show/174143979)
